### PR TITLE
New estimation with fixed car time and cost

### PR DESCRIPTION
**Pull request by atte-wsp**
New estimation and calibration to fix price sensitivity of model system. Changes were made trip for purposes: hw, hs, hu, ho, wo, oo, hwp and hop.

Model structure and coefficients are reported here:
https://hslfi.sharepoint.com/:f:/r/sites/ext-helmet_kehi_40/Jaetut%20asiakirjat/6%20Analyysit/Mallit/HELMET40_S2020_01?csf=1&web=1&e=I53bZV

### CODE REVIEW

File: Scripts/parameters.py [hunk 1]
```diff
@@ -2002,182 +1994,182 @@
...
-            "constant": (0.151688898 * (8.71611785 - 0.209), 
-                         0.151688898 * (-9.326014274 + 0.711)),
+            "constant": (0.157371648547 * (8.33856436370 - 0.380), 
+                         0.157371648547 * (-9.21721208402 - 0.258)),
             "generation": {
-                "cbd": 0.4,
-                "helsinki_other": 0.1,
-                "espoo_vant_kau": -0.7,
+                "cbd": 0.2,
+                "helsinki_other": 0.0,
```
Comment (zptro) [inline Scripts/parameters.py:2070]: Remove?

File: Scripts/parameters.py [hunk 2]
```diff
@@ -2002,182 +1994,182 @@
...
         "transit": {
-            "constant": (0.539979474415 * (5.13091589060 - 0.134), 
-                         0.539979474415 * (4.98462338486 - 0.476)),
+            "constant": (0.522036062262 * (6.72180796903 + 0.158), 
+                         0.522036062262 * (6.67197643351 - 0.399)),
             "generation": {
-                "cbd": 0.1,
-                "helsinki_other": 0.1,
+                "cbd": 0.0,
+                "helsinki_other": 0.0,
```
Comment (zptro) [inline Scripts/parameters.py:2016]: Remove?

File: Scripts/parameters.py [hunk 3]
```diff
@@ -1540,203 +1541,194 @@
...
-                "service": numpy.exp(1.54928000612),
-                "shops_cbd": numpy.exp(3.48055138106),
-                "shops_elsewhere": numpy.exp(3.30123960735),
-                "own_zone": numpy.exp(6.62440257143),
+                "workplaces": numpy.exp(.400884483384),
+                "service": numpy.exp(1.58922222310),
+                "shops_cbd": numpy.exp(3.58798748920),
+                "shops_elsewhere": numpy.exp(3.36839356543),
+                "own_zone": (numpy.exp(6.94153078815),
+                             numpy.exp(6.94153078815)),
```
Comment (zptro) [inline Scripts/parameters.py:1599]: These are the same so no need to have two different parameters

File: Scripts/parameters.py [hunk 4]
```diff
@@ -1275,211 +1274,213 @@
...
             },
             "size": {
                 "population_other": 1,
-                "population_own": numpy.exp(1.205650309),
-                "service_other": numpy.exp(2.384174142),
-                "service_own": numpy.exp(3.183207081),
-                "shops": numpy.exp(2.998477655),
-                "cbd": (numpy.exp(7.813868577), numpy.exp(4.709424332)),
+                "population_own": (numpy.exp(1.36533858970),
+                                   numpy.exp(1.36533858970)),
```
Comment (zptro) [inline Scripts/parameters.py:1477]: These are the same

File: Scripts/parameters.py [hunk 5]
```diff
@@ -1275,211 +1274,213 @@
...
+                "service_other": numpy.exp(2.39351567744),
+                "service_own": (numpy.exp(3.35003762339),
+                                numpy.exp(3.35003762339)),
```
Comment (zptro) [inline Scripts/parameters.py:1480]: These are the same

File: Scripts/parameters.py [hunk 6]
```diff
@@ -1095,71 +1095,71 @@
...
+                                   numpy.exp(0.906947845409)),
                 "workplaces_other": 1,
             },
         },
         "transit": {
             "attraction": {
-                "cbd": (0.483086068108, 0.483086068108),
-                "helsinki_other": (0.2, 0.2),
+                "cbd": (0.491418509053, 0.491418509053),
+                "helsinki_other": (0.00, 0.00),
```
Comment (zptro) [inline Scripts/parameters.py:1123]: Remove?

File: Scripts/parameters.py [hunk 7]
```diff
@@ -1095,71 +1095,71 @@
...
             },
         },
         "walk": {
             "attraction": {
-                "own_zone_area_sqrt": (-2.14509122337, -2.07210556201),
+                "own_zone_area_sqrt": (-2.15046600566, -2.08475720417),
             },
             "impedance": {},
             "log": {
-                "dist": -2.66058468415 + 0.1,
```
Comment (zptro) [inline Scripts/parameters.py:1157]: Is some kind of calibration removed?

Comment (atte-wsp) [reply Scripts/parameters.py:1157]: I removed all calibration parameters for calibrated trip purposes. I could not find arguments to return this calibration, so decided to left it out.

**Comment by atte-wsp** [comment]
Cost elasticity for trip purpose "ho" is still much lower than in previous Helmet versions. I will still look into this issue, if I could find an explanation.

**Comment by atte-wsp** [comment]
Trip purpose 'ho' has large alternative-speciﬁc constants for each mode and Dcoeff is exceptionally low, which means than destination utility (impedance, costs) has low impact on mode choice. The mode specific constants are correlated, which might cause this issue.

Could removing constants be one option? Or trying to remove the separation pks and kehys from these variables.

**Comment by atte-wsp** [comment]
> Trip purpose 'ho' has large alternative-speciﬁc constants for each mode and Dcoeff is exceptionally low, which means than destination utility (impedance, costs) has low impact on mode choice. The mode specific constants are correlated, which might cause this issue.
> 
> Could removing constants be one option? Or trying to remove the separation pks and kehys from these variables.

**Comment by zptro** [comment]
> 
> 
> Trip purpose 'ho' has large alternative-speciﬁc constants for each mode and Dcoeff is exceptionally low, which means than destination utility (impedance, costs) has low impact on mode choice. The mode specific constants are correlated, which might cause this issue.
> 
> Could removing constants be one option? Or trying to remove the separation pks and kehys from these variables.

So what this basically means is that generalised cost has a small impact on mode choice, but a larger impact on destination choice for these kind of trips. This is exactly what is observed in Stockholm and Gothenburg when implementing congestion charges.

**Review by zptro — APPROVED** [review]

**Comment by atte-wsp** [comment]
Sounds that this might no be an issue then. Maybe we can merge these changes and write this observation to model report for future consideration.

**Pull request merged**